apps/site/assets/ts/schedule/components/Map.tsx

```diff
@@ -194,11 +196,13 @@ export const reducer = (
 };
 
 export default ({
+  // Data has markers and stop_markers, and they are identical?
```

Comment:
@hannahpurcell is this a `TODO` to look into this? I don't think we do a lot of code TODOs, may be best to leave it as is or throw a tech-debt ticket in the backlog to investigate this as a later date (if this question is still unresolved)